dev/com.ibm.ws.cxf.client/src/com/ibm/ws/cxf/client/component/AsyncClientRunnableWrapperManager.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018 IBM Corporation and others.
+ * Copyright (c) 2019 IBM Corporation and others.
```

Comment:
I'm not sure what the copyright rules are for _moving_ a file, but I'd go with `2018-2019` just to be on the safe side.  Can you make the same change to `AsyncClientRunnableWrapper.java`?